snuba/consumers/consumer_builder.py

```diff
@@ -264,7 +262,7 @@ def build_streaming_strategy_factory(
                 processor,
                 self.consumer_group,
                 logical_topic,
-                self.__validate_schema,
+                float(get_config("validate_schema", 0) or 0.0),
```

Comment:
Let's support controlling the schema validation rate for each consumer independently?
```suggestion
                float(get_config(f"validate_schema_{storage_key.value}", 0) or 0.0),
```